bert/common/train.py

```diff
@@ -0,0 +1,285 @@
+import math
+import time
+
+import torch
+from torch.distributed import all_reduce, get_rank, get_world_size
+from tqdm import tqdm
+
+from zero.common.utils import CONFIG, AsyncMemoryMonitor, print_log, get_tflops
+
+
+def _train(epoch, rank, world_size, train_dataloader, model, criterion, optimizer, lr_scheduler, scaler, mem_monitor):

```

Comment:
Can you use typing hit for args?

Reply:
I will clean all codes from zero benchmark and condense them into example style after verifying the correctness. There are many designs I use now is for benchmark of different sources. They are unnecessary to colossalai-only demo.